opsdroid/events.py

```diff
@@ -82,8 +87,11 @@ def __init__(self, text, user, room, connector, raw_message=None):
         Logs response and response time in OpsDroid object stats.
         """
         opsdroid = get_opsdroid()
-        response = copy(self)
-        response.text = text
+        if isinstance(response_event, str):
+            response = copy(self)
```

Comment:
I think this should be `response = Message(response_event)` no? Are we not going to assume that a textual response is always a `Message`?

Reply:
Yes, I think that was what I was going for in a more round about way.

Reply:
If you go with this approach be sure to copy the other information like connector and room from `self`. The `copy` method is being used here to do that in a lazy way.